src/components/CorporateCampaign/CampaignProgressBar.vue

```diff
@@ -47,6 +47,32 @@
 			</kv-ui-button>
 		</div>
 	</div>
+	<div
+		v-else
+		class="
+			tw-rounded
+			tw-bg-brand-50
+			tw-w-full
```

Comment:
In response to your question about the button width, you could just only apple `tw-w-full` in mobile view, so the button doesn't stretch. Then the decision will have to be made on where the button sits (left, center, right) in desktop view.